chat-plugins/trivia.js

```diff
@@ -198,7 +198,7 @@ var Trivia = (function () {
 	};
 
 	Trivia.prototype.startGame = function (output) {
-		if (this.phase !== 'signup') return output.sendReply("There is no trivia game in its signup phase.");
+		if (this.phase !== 'signup') return output.sendReply("There is no trivia game to join.");
```

Comment:
Since this is ostensibly the function to start the game, why does the message say there is no trivia game to *join*?